### PR TITLE
Setting up benchmarks

### PR DESCRIPTION
**Pull request by JSOBot**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            value: '$.repository.name',
+            expressionType: 'JSONPath', //Optional, defaults to JSONPath
+            regexpFilter: '', //Optional, defaults to empty string
+            defaultValue: '' //Optional, defaults to empty string
+        ]
+     ],
+
+     causeString: 'Triggered on $comment',
+
+     token: "",
```
Comment (MonssafToukal) [inline Jenkinsfile:52]: Missing token name

**Comment by MonssafToukal** [comment]
It seems that the `benchmark/Project.toml` is missing as well.

**Comment by tmigot** [comment]
> It seems the `benchmark/Project.toml` is missing as well.

and `benchmarks.jl`.

**Comment by MonssafToukal** [comment]
We should probably close this PR as the files are now outdated.

**Comment by dpo** [comment]
@MonssafToukal could you open an updated pr?

**Pull request closed without merging**